dvc/repo/update.py

```diff
@@ -0,0 +1,16 @@
+def update(self, target):
+    from dvc.stage import Stage
+
+    stage = Stage.load(self, target)
+    if not stage.is_repo_import:
+        return
+
+    with self.state:
+        locked = stage.locked
+        stage.locked = False
+        stage.deps[0].erepo.rev_lock = None
```

Comment:
it should be part of the next call I think. update itself should not care about some internal very specific to the stage stuff.